Playbooks/Enrich-MalwareBazaar/readme.md

```diff
@@ -0,0 +1,32 @@
+# Enrich-MalwareBazaar
+
+Add information from [MalwareBazaar](https://bazaar.abuse.ch/api/) aka abuse.ch to a Sentinel Incident
+
+## Quick Deployment
+**Deploy with incident trigger** (recommended)
+
+After deployment, attach this playbook to an **automation rule** so it runs when the incident is created.
+
+[Learn more about automation rules](https://docs.microsoft.com/azure/sentinel/automate-incident-handling-with-automation-rules#creating-and-managing-automation-rules)
+
+[![Deploy to Azure](https://aka.ms/deploytoazurebutton)](https://portal.azure.com/#create/Microsoft.Template/uri/https%3A%2F%2Fraw.githubusercontent.com%2FAzure%2FAzure-Sentinel%2Fmaster%2FPlaybooks%2FEnrich-MalwareBazaar%2FPlaybook%2Fazuredeploy.json)
+[![Deploy to Azure Gov](https://aka.ms/deploytoazuregovbutton)](https://portal.azure.us/#create/Microsoft.Template/uri/https%3A%2F%2Fraw.githubusercontent.com%2FAzure%2FAzure-Sentinel%2Fmaster%2FPlaybooks%2FEnrich-MalwareBazaar%2FPlaybook%2Fazuredeploy.json)
+
+## Prerequisites
+
+* No API key for malwarebazaar
```

Comment:
are you trying to say here that APi key is not needed ? If yes then , 
Can you please mention like this ,  Api is not required for malware bazar ? or either you can remove the complete line .